packages/store/src/StoreView.sol

```diff
@@ -2,74 +2,42 @@
 pragma solidity >=0.8.0;
 
 import { SchemaType } from "./Types.sol";
-import { IStore, IStoreHook } from "./IStore.sol";
+import { IStoreHook } from "./IStore.sol";
 import { StoreCore } from "./StoreCore.sol";
 import { Schema } from "./Schema.sol";
+import { Store } from "./Store.sol";
 
 // Not abstract, so that it can be used as a base contract for testing and wherever write access is not needed
-contract StoreView is IStore {
+contract StoreView is Store {
   error StoreView_NotImplemented();
 
-  constructor() {
-    StoreCore.initialize();
-  }
-
-  function getSchema(uint256 table) public view virtual returns (Schema schema) {
-    schema = StoreCore.getSchema(table);
-  }
-
   function registerSchema(uint256, Schema) public virtual {
```

Comment:
I'd add `override` to the relevant `StoreView` methods

Reply:
which methods to you mean in particular? (`StoreView` just "implements" the `IStore` functions missing from `Store`, but doesn't really override anything)

Reply:
Yeah that's kinda what I meant - to use override for implemented interface functions. https://github.com/ethereum/solidity/issues/8281 has some discussion on that, and I'm generally in favor of using `overrides` to indicate which functions come from interfaces and not just other contracts. But it's really fine either way, up to you.

Reply:
Oh I see, after reading through the discussion in https://github.com/ethereum/solidity/issues/8281 I'd tend to not doing this, at least not for now - there are good arguments for both sides, but since we don't do it currently with other interfaces in the MUD codebase, I'd aim for consistency in this case.